sign_test.go

```diff
@@ -201,27 +117,29 @@ func TestVerify_rust(t *testing.T) {
 	sigin := [64]byte{}
 	copy(sigin[:], sighex)
 
-	sig := &Signature{}
+	sig := &schnorrkel.Signature{}
 	err = sig.Decode(sigin)
 	require.NoError(t, err)
 
-	transcript := NewSigningContext(SigningContext, msg)
+	transcript := schnorrkel.NewSigningContext(signingContext, msg)
 	ok, err := pub.Verify(sig, transcript)
 	require.NoError(t, err)
 	require.True(t, ok)
 }
 
 func TestVerify_PublicKeyAtInfinity(t *testing.T) {
+	publicKeyAtInfinity := r255.NewElement().ScalarBaseMult(r255.NewScalar())
```

Comment:
Since making this a local var does remove the need for the global var, I think we can just keep this test as is (unless there is another reason this was updated)